frontend/src/router/index.ts

```diff
@@ -1082,6 +1106,27 @@ router.beforeEach((to, from, next) => {
     return;
   }
 
+  if (connectionSlug) {
+    const [instanceSlug, instanceId, databaseSlug, databaseId] =
+      connectionSlug.split("_");
+    store
+      .dispatch("sqlEditor/setConnectionFromRoute", {
```

Comment:
let's call it fetchConnectionByInstanceIdAndDatabaseId

Reply:
agree